pkg/controller/manager/manager_controller.go

```diff
@@ -674,9 +674,6 @@ func (r *ReconcileManager) processContrailCNIs(manager *v1alpha1.Manager) error
 			oldContrailCNI.ObjectMeta = v1.ObjectMeta{
 				Namespace: manager.Namespace,
 				Name:      *existingContrailCNI.Name,
-				Labels: map[string]string{
-					"contrail_cluster": "cluster1",
-				},
```

Comment:
Now it will delete any ContrailCNI object in the namespace. Shouldn't it delete only the one in the same "contrail_cluster" as the current Manager?

Reply:
Done - for all resources